resources/resources.qrc

```diff
@@ -51,6 +51,8 @@
     <file>icons/error.png</file>
     <file>icons/unparseable.png</file>
     <file>icons/editIcon.png</file>
+    <file>icons/collapse.png</file>
```

Comment:
does not build currently -- must include collapse.png and expand.png in icons folders